src/common/options/global.yaml.in

```diff
@@ -5365,6 +5365,30 @@ options:
   desc: Enable health indication when spurious read errors are observed by OSD
   default: true
   with_legacy: true
+- name: bluestore_slow_ops_warn_lifetime
+  type: uint
+  level: advanced
+  desc: Duration of slow op warning of bluestore should persist since last occurence
+  default: 30
+  with_legacy: true
+- name: bluestore_slow_ops_warn_threshold
+  type: uint
+  level: advanced
+  desc: Last maximum number of slow op event considered
```

Comment:
IMO this isn't very informative. Not to mention that it's unclear to me what "Last(?)  maximum number" really means here...
The same for bdev_stalled_read_warn_threshold